lib/managers/files.js

```diff
@@ -560,6 +560,22 @@ Files.prototype.deleteMetadata = function(fileID, scope, template, callback) {
 };
 
 /**
+ * Retrieves a file that has been moved to the trash.
+ *
+ * API Endpoint: '/files/:fileID/trash'
+ * Method: GET
+ *
+ * @param {string} fileID - The ID of the file being requested
+ * @param {Function} callback - Called with nothing if successful, error otherwise
+ * @returns {void}
+ */
+Files.prototype.getTrashedFile = function(fileID, callback) {
```

Comment:
I think this API endpoint accepts the fields parameter, so we probably need to include the ability to pass qs here.